docs/htdocs/info/docs/tools/vep/script/vep_other.html

```diff
@@ -539,9 +539,9 @@ <h2 id="colocated">Existing or colocated variants</h2>
   <br />
   <h3 id="freq">Frequency data</h3>
 
-  <p> In addition to identifying known variants, VEP also reports allele frequencies for input alleles from major genotyping projects (<a href="vep_options.html#opt_af_1kg">1000 genomes</a>, <a href="vep_options.html#opt_af_esp">ESP</a> and <a href="vep_options.html#opt_af_gnomad">gnomAD</a>). VEP's cache currently contains only frequency data for alleles that have been submitted to dbSNP or are imported via <a href="/info/genome/variation/species/sources_documentation.html">another source</a> into the Ensembl variation database. This means that until gnomAD's full data set is submitted to dbSNP and incorporated into Ensembl, the frequency for some alleles may be missing from VEP's cache data. </p>
+  <p> In addition to identifying known variants, VEP also reports allele frequencies for input alleles from major genotyping projects (<a href="vep_options.html#opt_af_1kg">1000 genomes</a>, <a href="vep_options.html#opt_af_gnomade">gnomAD exomes</a> and <a href="vep_options.html#opt_af_gnomadg">gnomAD genomes</a>). VEP's cache currently contains only frequency data for alleles that have been submitted to dbSNP or are imported via <a href="/info/genome/variation/species/sources_documentation.html">another source</a> into the Ensembl variation database. This means that until gnomAD's full data set is submitted to dbSNP and incorporated into Ensembl, the frequency for some alleles may be missing from VEP's cache data. </p>
 
-  <p> To access the full gnomAD data set, it is possible to use VEP's custom annotation feature to retrieve the frequency data directly from the gnomAD VCF files; see <a href="vep_example.html#gnomad">instructions here</a>. </p>
+  <p> To access the full gnomAD data set, it is possible to use VEP's custom annotation feature to retrieve the frequency data directly from the gnomAD VCF files; see <a href="vep_example.html#gnomade">instructions here</a>. </p>
```

Comment:
This section is still named `gnomad`:
```suggestion
  <p> To access the full gnomAD data set, it is possible to use VEP's custom annotation feature to retrieve the frequency data directly from the gnomAD VCF files; see <a href="vep_example.html#gnomad">instructions here</a>. </p>
```